pjmedia/src/pjmedia/sdp_neg.c

```diff
@@ -1623,6 +1955,11 @@ PJ_DEF(pj_status_t) pjmedia_sdp_neg_negotiate( pj_pool_t *pool,
                                neg->neg_local_sdp, neg->neg_remote_sdp,
                                &answer);
         if (status == PJ_SUCCESS) {
+
+#if !PJMEDIA_SDP_NEG_ANSWER_SYMMETRIC_PT
+            validate_pt(pool, neg, answer);
```

Comment:
Should we prioritise answering with symmetric PT, or making sure that the mapping is valid?

IMO, we should try to answer with symmetric PT, but if it causes the mapping to be invalid, then we have no choice but use asymmetric PT.